sample/WebHook-Generic-CSharp/run.csx

```diff
@@ -4,13 +4,18 @@ using System;
 using System.Net;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
 
 public static async Task<object> Run(HttpRequestMessage req, TraceWriter log)
 {
     string json = await req.Content.ReadAsStringAsync();
     Payload payload = JsonConvert.DeserializeObject<Payload>(json);
 
-    return req.CreateResponse(HttpStatusCode.OK, $"Value: {payload.Value}");
+    JObject body = new JObject()
+    {
+        { "result", $"Value: {payload.Value}" }
+    };
+    return req.CreateResponse(HttpStatusCode.OK, body, "application/json");
```

Comment:
I like leaving that up to the user. In fact, using CreateResponse with just the HTTP status and the payload (without explicitly setting the content type) should allow content negotiation to handle the content type used, taking accept header and other factors into consideration.